realm/realm-library/src/main/java/io/realm/DynamicRealm.java

```diff
@@ -279,6 +279,22 @@ static DynamicRealm createInstance(RealmConfiguration configuration) {
     }
 
     /**
+     * Returns a distinct set of objects from a specific class that fulfill the multiple fields
+     * condition. If the result is sorted, the first object of a column will be returned in case of
+     * multiple occurrences, otherwise it is undefined which object is returned.
+     *
+     * @param className the Class to get objects of.
+     * @param fieldNames the multiple fields.
```

Comment:
This could use a rephrase, doesn't really provide any meaningful info. Also needs to be rewritten when splitting the var args.